src/components/frame/TokenActivation/StakesTab/StakesListItem.tsx

```diff
@@ -38,9 +39,15 @@ const StakesListItem = ({
   });
 
   const txHash = data?.getColonyActionByMotionId?.items[0]?.id ?? '';
+  const location = useLocation();
+  const replace = location.pathname.indexOf('/tx') !== -1;
```

Comment:
How about a more descriptive name for this variable, e.g. `isMotionPage`? I feel like it'd make it easier to understand at first glance